src/components/Resource/ResourceCreate.tsx

```
@@ -114,86 +151,130 @@
-    });
-  };
+  const handleSubmit = async (form: ResourceData) => {
+    setIsLoading(true);
+
+    const resourceData = {
+      status: "PENDING",
+      category: form.category,
+      sub_category: form.sub_category?.toString(),
+      origin_facility:
+        form.origin_facility_object?.id || String(props.facilityId),
+      approving_facility: (form.approving_facility_object || {}).id,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)